examples/terminal/templates/error.html

```diff
@@ -10,7 +10,7 @@
 
   <title>{% block title %}{{page_title | e}}{% endblock %}</title>
 
-  {% block favicon %}<link rel="shortcut icon" type="image/x-icon" href="/static/favicons/favicon.ico">{% endblock %}
+  {% block favicon %}<link rel="shortcut icon" type="image/x-icon" alt="icon for shortcut" href="/static/favicons/favicon.ico">{% endblock %}
```

Comment:
While it might be less impactful in a short description like this, I do recommend keeping at least sentence casing like we do for other user-facing labels (ie. tool tips or button names).

```suggestion
  {% block favicon %}<link rel="shortcut icon" type="image/x-icon" alt="Shortcut icon" href="/static/favicons/favicon.ico">{% endblock %}
```